src/main/java/uk/gov/hmcts/darts/arm/service/impl/ArmResponseFilesProcessorImpl.java

```diff
@@ -111,6 +108,18 @@ private void initialisePreloadedObjects() {
         userAccount = userIdentity.getUserAccount();
     }
 
+    @SuppressWarnings("java:S3655")
+    @Transactional
+    public void processInputUploadFile(Integer eodId) {
```

Comment:
Transactional methods need to be called externally from the class as it only triggers the transactional aspect when it hits the proxy object